src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -362,8 +427,10 @@ HttpStatusCode mongoQueryContext
   /* In the case of empty response, if only generic processing is needed */
   if (rawCerV.size() == 0)
   {
+    LM_T(LmtForward, ("Calling registrationsQuery I"));
     if (registrationsQuery(requestP->entityIdVector, requestP->attributeList, &crrV, &err, tenant, servicePathV, 0, 0, false))
     {
+      // crrVectorPresent("registrationsQuery I", crrV);
```

Comment:
I kept (commented out) the calls to the debug functions `crrVectorPresent` and `cerVectorPresent`, and the functions themselves.
Will be needed when a better version of forwarding are implemented, possibly right after finishing this PR.

Reply:
Ok, but put a FIXME mark above the line to locate it easyly in the future for cleanup.

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed